### PR TITLE
update release guide for 4.13.0.Final and 12.13.0.GA

### PR DESCRIPTION
**Pull request by sbouchet**
Signed-off-by: Stephane Bouchet <sbouchet@redhat.com>

### CODE REVIEW

File: release_guide/12/1_Staging_preparation.adoc [hunk 1]
```diff
@@ -127,8 +127,8 @@ https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Devstudio/view
 
 Currently:
 
-version_jbt_IS=4.6.0.Final
-version_ds_IS=12.0.0.GA
+ version_jbt_IS=4.6.0.Final
```
Comment (jeffmaury) [inline release_guide/12/1_Staging_preparation.adoc]: Latest version is 4.11.0.Final see https://download.jboss.org/jbosstools/photon/stable/updates/integration-stack/
For Studio 12.11.0.Final see https://devstudio.redhat.com/12/stable/updates/integration-stack

File: release_guide/12/4_Release_devstudio.adoc [hunk 2]
```diff
@@ -271,14 +269,17 @@ Before notifying team of release, must check for obvious problems. Any failure t
 
 1. Download JBDS installer from https://devstudio.redhat.com/12/stable/builds/
 2. Install via UI or headlessly with
-    java -jar devstudio-*.jar -console -options /dev/null # **
+
+ java -jar devstudio-*.jar -console -options /dev/null # **
```
Comment (jeffmaury) [inline release_guide/12/4_Release_devstudio.adoc]: Should be codereadystudio-

File: release_guide/12/1_Staging_preparation.adoc (hunk 1 above)
Comment (sbouchet) [reply release_guide/12/1_Staging_preparation.adoc]: fixed.

File: release_guide/12/4_Release_devstudio.adoc (hunk 2 above)
Comment (sbouchet) [reply release_guide/12/4_Release_devstudio.adoc]: fixed.

**Pull request merged**